.github/workflows/npm-eslint-plugin-meteor.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         working-directory: npm-packages/eslint-plugin-meteor
     strategy:
       matrix:
-        node-version: [12.x, 14.x]
+        node-version: [14.x, 18.x]
```

Comment:
Given that Meteor 3 will be on Node 20, we need to update these accordingly.

Reply:
Agree! I will create a PR with 14.21.4.7 with those change as well